src/test/java/sql/basic/SqlTest.java

```diff
@@ -143,6 +143,13 @@ private void testBatch(int n){
         //Check that records were deleted
         Assert.assertEquals(0,reader.all().count());
         
+        creator.from(created.stream());
+        
```

Comment:
Check that something was in fact created and that size of adapter pool is non-empty: this test checks out if no object is created (maybe failed silently) and deletion fails to perform, which is not what we want to check.